### ISSUE TITLE
Browse mode: NvDA announces characters entered

### ISSUE THREAD

**Issue by josephsl** [dictationbridge/dictationbridge-nvda#10]
Hi,

Consider the following scenario: a user is watching videos from places such as YouTube where shortcut keys were defined to manipulate video controls such as rewind and muting. If these commands are pressed, NVDA will announce them with DB turned on even though the user told NVDA to not announce typed characters.

Thanks.

**Comment by derekriemer** [comment]
I'm not sure this is a bug when in a text editor because speech isn't typing. However, in a browser and not an edit field, sure.
